src/chrome/content/rules/Groupon.com.xml

```diff
@@ -18,25 +11,23 @@ Fetch error: http://touch2.groupon.com/ => https://touch2.groupon.com/: (28, 'Co
 
 	Nonfunctional subdomains:
 
+		- @ ⁴
+		- www ⁴
```

Comment:
I can not confirm the 403 on them. Maybe you have to use a static resource as test url.